elasticsearch/utils/allocate-stale-primary

```diff
@@ -15,36 +15,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 set -euo pipefail
-source es_util_env
 
-shard=0
-allow_primary=true
+if [ -z "${OC_CMD:-''}" ] ; then
```

Comment:
how would we use `OC_CMD`? What would it be set to?

Reply:
you don't need the `''` - you can just use `[ -z "${OC_CMD:-}" ]`

Reply:
For example, if you had a copy of the script outside of the pod, on the master host, you could run
```
OC_CMD="oc exec -c elasticsearch $espod" ./allocate_stale_primary index ...
```

Reply:
I have found that some of these utils are useful outside the context of the pod.  This allows you to either use on pod or extract and use on host like:

``
DC_NAME=logging-es-data-master-f26an8nv OC_CMD="oc exec -c elasticsearch $pod --" SHARD=0 ./allocate-stale-primary .searchguard
```
This allows use without needing to modify it